packages/console/src/cloud/pages/Onboard/index.tsx

```diff
@@ -34,6 +37,14 @@ const Onboard = () => {
             conditional(questionnaire && <About />) ?? <Navigate replace to={welcomePathname} />
           }
         />
+        <Route
+          path={OnboardPage.SignInExperience}
+          element={
+            conditional(questionnaire && <SignInExperience />) ?? (
+              <Navigate replace to={welcomePathname} />
+            )
```

Comment:
`questionnaire ? <SignInExperience /> : <Navigate replace to={welcomePathname} />`